src/app/dataset-view/dataset-view.interface.ts

```diff
@@ -6,3 +15,21 @@ export enum DatasetViewTypeEnum {
     discussions = "discussions",
     history = "history",
 }
+
+export interface DatasetViewContentInterface {
```

Comment:
Perhaps `TableContentInterface` would be a more fitting name? `DatasetView` is how we call the whole multi-tab view component.